make/buf/scripts/release.bash

```diff
@@ -140,6 +140,11 @@ for os in Darwin Linux; do
   done
 done
 
+apt-get -y update
+apt-get -y autoremove
+apt-get clean
+apt-get install -y zip
+
```

Comment:
This is my fault, lol, but yeah, given that we want to make sure `release.bash` runs locally on all machines, instead of installing here, we should create a Dockerfile instead.